project_charters/opentimelineio-charter.md

```diff
@@ -1,6 +1,3 @@
-<!-- SPDX-License-Identifier: CC-BY-4.0 -->
-<!-- Copyright Copyright Contributors to the [COMMUNITY NAME] Project -->
-
```

Comment:
@jmertic is this important to retain? I didn't see it in any of the other charters, but it is in the template.

Reply:
It's fine to remove